rasa_core/policies/keras_policy.py

```diff
@@ -240,10 +264,11 @@ def load(cls, path: Text) -> 'KerasPolicy':
                 meta = json.loads(utils.read_file(meta_path))
```

Comment:
since we persist the whole config in meta, let's pass the rest of the meta to init as well for consistency

Reply:
Now I persist in `keras_policy.config.pkl` rather than in meta.